dradis-nipper.gemspec

```diff
@@ -1,31 +1,28 @@
-$:.push File.expand_path("lib", __dir__)
+$:.push File.expand_path('lib', __dir__)
 
 # Maintain your gem's version:
-require "dradis/nipper/version"
+require 'dradis/plugins/nipper/version'
+version = Dradis::Plugins::Nipper::VERSION::STRING
 
 # Describe your gem and declare its dependencies:
 Gem::Specification.new do |spec|
-  spec.name        = "dradis-nipper"
-  spec.version     = Dradis::Nipper::VERSION
-  spec.authors     = [""]
-  spec.email       = [""]
-  spec.homepage    = "TODO"
-  spec.summary     = "TODO: Summary of Dradis::Nipper."
-  spec.description = "TODO: Description of Dradis::Nipper."
-  spec.license     = "MIT"
+  spec.platform    = Gem::Platform::RUBY
+  spec.name        = 'dradis-nipper'
+  spec.version     = version
+  spec.authors     = ['Daniel Martin']
+  spec.email       = ['etd@nomejortu.com']
+  spec.homepage    = 'http://dradisframework.org'
+  spec.summary     = 'Nipper upload add-on for Dradis Framework.'
+  spec.description = 'This add-on allows you to upload and parse reports from Nipper.'
+  spec.license     = 'GPL-2'
```

Comment:
Can you please use the standard values we use for other gems (see Okta, or Azure)